docs/index.markdown

```diff
@@ -0,0 +1,160 @@
+
+# Prerequisites
+
+In order to function properly, the Blossom extension requires the Swarm extension to be installed and configured.
+
+Open the [Google Web Store page](https://chrome.google.com/webstore/detail/ethereum-swarm-extension/afpgelfcknfbbfnipnomfdbbnbbemnia){:target="\_blank"} and install the Swarm extension first.
+
+The Swarm extension provides security context for dApps that are loaded from bzz links.
+
+## Swarm Extension Configuration
+
+Configure your Swarm extension, by setting the correct URL of your Bee node. Later the Blossom will use this URL as storage.
+
+If you are running a local Bee node on the default port, then no changes are needed. Otherwise update it to correct URL.
+
+![Swarm Extension URL config](/assets/img/swarm-extension-bee-url.png){:width="70%"}
+
+If you don't have a running Bee node, you can use the FDS gateway:
+
+```
+https://bee-1.fairdatasociety.org
+```
+
+# Installation
+
+Now when the Swarm extension is installed and configured, to install the Blossom extension open the [Google Web Store page](https://chrome.google.com/webstore/detail/blossom/caedjloenbhibmaeffockkiallpngmmd){:target="\_blank"}.
+
+## Creating an account
+
+To use the Blossom extension, the user must have an account. To create one, click on the extension icon and select the `LOGIN OR REGISTER` option.
+
+![Blossom Extension after installation](/assets/img/blossom-not-logged-in.png){:width="50%"}
+
+A new page will open where you can choose whether you want to create a portable or local account.
+
+![Blossom Welcome Screen](/assets/img/blossom-welcome-page.png)
+
+Portable accounts are encrypted and stored in the Swarm network. That way the user can access them on any device using the password they provided in registration process.
+
+There is also an option to create a local account, which is not stored in the Sworm network. Instead, it's stored only on the user's local machine.
+
+### Creating a portable account
+
+To create a portable account click on the `REGISTER` button. In the next screen, first choose the blockchain you want to register the account to. At the moment there are only `FDP-Play` and `Görli` blockchains.
+
+> **_NOTE:_** For FDP-Play option, the fdp-play environment must be running locally. Check [its repo](https://github.com/fairDataSociety/fdp-play){:target="\_blank"} for more info
+
+> **_NOTE:_** You can add a custom blockchain to the list, but in order to register an account there it must have all the required smart contracts deployed. Check the [Adding a custom blockchain network](#adding-a-custom-blockchain-network) section for more information.
+
+Choose a username and password for your account, and click the `REGISTER` button. If the username is available and password is secure enough, a new screen will open. There you can choose to generate a new seed phrase for your account which is the `NEW ACCOUNT` option, or you can provide an existing seed phrase - `EXISTING ACCOUNT`.
```

Comment:
Make sure the password is very strong as it cannot be changed and it protects access to your data.